app/scripts/modules/core/src/pipeline/executions/execution/Execution.tsx

```diff
@@ -379,7 +413,42 @@ export class Execution extends React.Component<IExecutionProps, IExecutionState>
               </Tooltip>
             )}
           </div>
+
+          <div className="execution-parameters-button">
```

Comment:
We have a `CollapsibleSection` component to show/hide sections and as a bonus includes an animated chevron

See here for an example usage: 
https://github.com/spinnaker/deck/blob/6e29dacbac867900087c2c169ee971ac084e963a/app/scripts/modules/core/src/presentation/forms/fields/FormikExpressionRegexField.tsx#L63-L68

- `heading` can be a string (or a function component).

note: I suspect this component cannot be used today to also collapse the Details panel because of the unique styling requirements.

Reply:
It's not too easy to wire this one up with pinned parameters, since Pinned parameters are kind of "semi" expanded.